cves/kernel/CVE-2021-46283.yml

```diff
@@ -329,9 +352,12 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
+  discussed_as_security: false
+  any_discussion: false
+  note: |
+    Some dicussion was found in the bug report but none of it disagreed with the 
```

Comment:
see my comment on the discussion on your other CVE. similar changes could be applicable here. (tl;dr reword this)